numeric.rb

```diff
@@ -321,6 +321,27 @@ def numerator
   def denominator
     1
   end
+
+  with_yjit do
+    if Primitive.rb_builtin_basic_definition_p(:downto)
+      undef :downto
+
+      def downto(to) # :nodoc:
+        Primitive.attr! :inline_block, :c_trace
+
+        unless defined?(yield)
+          return Primitive.cexpr! 'SIZED_ENUMERATOR(self, 1, &to, int_downto_size)'
+        end
```

Comment:
Can you explain what this bit does? May be good to add a comment

Reply:
added a comment https://github.com/ruby/ruby/pull/12074/commits/300891fce49752130cfc57c7fded7356e0ee2ccd